### DISCUSSION TITLE
Welcome to OctoPrint-SpoolManager Discussions!

### DISCUSSION THREAD

**Discussion by dojohnso** [dojohnso/OctoPrint-SpoolManager#1] [Announcements]
## 👋 Welcome!
I'm using Discussions as a place to connect with other members of the community. I hope that you:
  * Ask questions you’re wondering about.
If you're unsure if something should be created as an Issue, ask here first.
Share ideas for new features.
Engage with other community members.
Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by Jadefunkel** [comment]
Hi, I have this newly installed and I have two instances running. I would love to see an option for the external database connection, so that both printers can share the same spool information. It is under development?
Is there something foreseen, to support multiple color changes for one print? As for Hueforge or other related prints. 
A lot of thanks for your effort! I already like this plugin very much!

**Comment by chemixt** [comment]
I love spool manager.  I have been using for few years now, but now I have another 3D printer, and I am trying to figure out how to keep track of my inventor and spool levels.  I noticed that the Export feature I have uese in the past is now no long present, (or am I missing something).  Can you tell me if I can export to CSV my spool data?
